src/Forms/FieldList.php

```diff
@@ -113,9 +107,9 @@ public function dataFields()
     }
 
     /**
-     * @return FormField[]
+     * @return array<FormField>|null
      */
-    public function saveableFields()
+    public function saveableFields(): ?array
```

Comment:
Should just return `array` due to the change in https://github.com/silverstripe/silverstripe-framework/pull/11235#discussion_r1599180952

Reply:
Done